resources/lang/en/auth.php

```diff
@@ -14,6 +14,6 @@
     */
 
     'failed' => 'These credentials do not match our records.',
-    'throttle' => 'Too many login attempts. Please try again in :seconds seconds.',
+    'throttle' => 'Too many Login attempts. Please try again in :seconds seconds.',
 
```

Comment:
probeer dit soort dingetjes eruit te halen bij de volgende commits

Reply:
Nogmaals. Nooit aan gezeten dus nooit gezien dat dit het geval is geweest.